app/views/categories/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<nav class="categories-nav">
+  <ul class="category-nav">
+    <li>
+      <%= link_to '<', payments_path, class: "arrow-left" %>
+    </li>
+    <li>
+      <h1 class="transf">Category Details</h1>
+    </li>
+    <li>
+      <%= button_to 'LOGOUT', destroy_user_session_path, method: :delete, class: "deletes" %>
+    </li>
+  </ul>
+</nav>
+<p style="color: green"><%= notice %></p>
+
+<div class="payments">
+  <div class="transaction">
+   <p>Total Transactions</p>
+   <p>$<%= @category.payments.sum(:amount) %></p>
+  </div>
+</div>
+
+<% @payments.each do |payment| %>
+  <div class="micons">
+    <div class="single">
+      <div>
+        <%= @category.icon %>
+      </div>
+      <div>
+        <p><%= payment.name %></p>
+        <p><%= payment.created_at.strftime("%Y/%m/%d %I:%M %p") %></p>  
+      </div>
+      <div>
+        <p>$<%= payment.amount %></p>
+      </div>
+    </div>
+  </div> 
+<% end %>
+
+<div class="payment-btn">
+  <%= link_to "ADD NEW TRANSACTION", new_payment_path(category_id: @category.id), class: 'expense-btn' %>
+</div>
```

Comment:
- [ ] Please ensure that the Button is styled and placed correctly on the page as shown in the Behance design template provided. Kindly take a look 👇 
## Behance Design
![image](https://user-images.githubusercontent.com/46839548/189116768-7f1093cc-6403-4f85-811d-d3c243b11cb8.png)
## Your Design
![image](https://user-images.githubusercontent.com/46839548/189116667-a3b9962f-b63e-4a1d-9de1-d7cef40e256b.png)